raycicmd/command_step.go

```diff
@@ -74,6 +74,20 @@ func (c *commandConverter) convert(id string, step map[string]any) (
 	if err != nil {
 		return nil, fmt.Errorf("map agent: %w", err)
 	}
+	concurrency_group, _ := stringInMap(step, "concurrency_group")
+	allowed_concurrency_group_prefixes := c.config.AllowConcurrencyGroupPrefixes
+	if concurrency_group != "" {
+		allowed_group := false
+		for _, group_prefix := range allowed_concurrency_group_prefixes {
+			if strings.HasPrefix(concurrency_group, group_prefix) {
+				allowed_group = true
+				break
+			}
+		}
```

Comment:
have a helper function that returns boolean for this? and add some unit tests for the helper function?